src/modules/aboutPage/index.tsx

```diff
@@ -22,7 +22,7 @@ const StyledListItem = styled(ListItem)({
   listStylePosition: 'inside',
 });
 
-const WhyWeList = [
```

Comment:
config constants usually named in uppercase WHY_WE_LIST

Reply:
fixed